### PR TITLE
Project Execution - H1

### PR DESCRIPTION
**Pull request by McDoxa**

### CODE REVIEW

File: project-javascript-h1-hof/Oluwaseyi.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <script>
+        //3. Higher Order Function (HOF)
+        const repeat = (func, param, times) =>{
+            for (let i=0; i<times; i++){
+                func(param);
+            }
+            return console.log('Run number: ' + times);
+        }
+        // defining the function 'func'.
+            const func = param => {};
```
Comment (basir) [inline project-javascript-h1-hof/Oluwaseyi.html]: No need to declare this function.

File: project-javascript-h1-hof/Oluwaseyi.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <title>Functions Benchmark</title>
+</head>
+<body>
+    <script>
+        //3. Higher Order Function (HOF)
+        const repeat = (func, param, times) =>{
+            for (let i=0; i<times; i++){
+                func(param);
+            }
+            return console.log('Run number: ' + times);
```
Comment (basir) [inline project-javascript-h1-hof/Oluwaseyi.html]: Move it into for loop to log each iteration into console.

File: project-javascript-h1-hof/Oluwaseyi.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        }
+        // defining the function 'func'.
+            const func = param => {};
+
+        //4.
+        function repeatDuration(repeat, times,func, param){
+            let startTime = new Date();
+            repeat(func, param, times);
+            let stopTime = new Date();
+            return console.log("Running time is " + (stopTime.getTime() - startTime.getTime()) + " milliseconds");
```
Comment (basir) [inline project-javascript-h1-hof/Oluwaseyi.html]: do not use `let` until you have to. No need to call `getTime()`

**Comment by McDoxa** [comment]
Changes have been effected. Thank you.

**Pull request merged**